ci/prepare.jl

```diff
@@ -10,3 +7,6 @@ Pkg.update()
 # Install dependencies:
 info("Pkg.add(PyCall)")
 Pkg.add("PyCall")
+
+# Show Julia information in verbose mode:
+versioninfo(true)
```

Comment:
It has to come after `Pkg.init()`.  Otherwise:
> ERROR: LoadError: Package metadata directory /builds/tkfpub/pyjulia-testing-template/.julia/v0.6/METADATA doesn't exist; run Pkg.init() to initialize it.

https://gitlab.com/tkfpub/pyjulia-testing-template/-/jobs/68814727